docs/resources/storage_volume_efficiency_policies.md

```diff
@@ -0,0 +1,123 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "netapp-ontap_volume_efficiency_policies Resource - terraform-provider-netapp-ontap"
+subcategory: "Storage"
+description: |-
+  VolumeEfficiencyPolicies resource
+---
+
+# netapp-ontap_volume_efficiency_policies (Resource)
+
+Create/Modify/Delete a VolumeEfficiencyPolicies resource
+
+### Related ONTAP commands
+```commandline
+* vol efficiency policy create
+* vol efficiency policy modify
+* vol efficiency policy delete
+```
+
+## Supported Platforms
+* On-perm ONTAP system 9.6 or higher
+* Amazon FSx for NetApp ONTAP
+
+## Example Usage
+
+```terraform
+resource "netapp-ontap_volume_efficiency_policies" "storage_volume_efficiency_policy" {
+  # required to know which system to interface with
+  cx_profile_name = "cluster4"
+  name = "ACC-import"
+  svm = {
```

Comment:
We use svm_name for all the resources.

Reply:
many other resources use this type as well